classes/shortcodes.php

```diff
@@ -216,20 +252,28 @@ function do_shortcode( $args ) {
 					switch ( $tax_format ) {
 						case 'csv':
 						default:
-							return ! empty( $term_names ) ? $args['before'] . implode( ', ', $term_names ) . $args['after'] : '';
+							return ! empty( $term_names ) ? $args['before']
+							                                . implode( ', ',
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Opening parenthesis of a multi-line function call must be the last content on the line

<!-- Comment by [SideCI](https://sideci.com) -->